pkgs/development/python-modules/vdf/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "vdf";
+  version = "3.1";
+
+  src = fetchPypi {
```

Comment:
Please fetch from the github repository and enable the python tests.

You can do so by overriding the `checkPhase` to run 
* https://github.com/ValvePython/vdf/blob/master/Makefile#L8

You'll also have to add the [testing deps](https://github.com/ValvePython/vdf/blob/master/requirements.txt) to `checkInputs`.